reason-react-native/src/apis/Style.md

```diff
@@ -4,4 +4,1298 @@ title: Style
+
+Use `elevation` style props for Android.
+
+#### `shadowColor`
+
+Accepts a `Color.t` (`string`).
+
+Sets the drop shadow color
+
+Only for
+
+- iOS
+
+You can see it as CSS `box-shadow` color
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow)
+
+#### `shadowOffset`
+
+Accepts an `offset`.
+
+Sets the drop shadow offset
+
+Only for
+
+- iOS
+
+You can see it as CSS `box-shadow` offsets
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow)
+
+#### `shadowOpacity`
+
+Accepts a `float`.
+
+Sets the drop shadow opacity (multiplied by the color's alpha component)
+
+Only for
+
+- iOS
+
+You can see it as CSS `box-shadow` color alpha value
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow)
+
+#### `shadowRadius`
+
+Accepts a `float`.
+
+Sets the drop shadow blur radius
+
+Only for
+
+- iOS
+
+You can see it as CSS `box-shadow` blur radius
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow)
+
+### Transform Style Props
+
+[Official documentation](https://facebook.github.io/react-native/docs/transforms)
+
+⚠️ _We only included `transform` prop as other have been deprecated._
+
+#### `transform`
+
+Accepts an array of `transform`.
+
+Keep in mind that order or transformation matters.
+
+- `perspective(~perspective=float)`
+- `rotate(~rotate=angle)`
+- `rotateX(~rotateX=angle)`
+- `rotateY(~rotateY=angle)`
+- `rotateZ(~rotateZ=angle)`
+- `scale(~scale=float)`
+- `scaleX(~scaleX=float)`
+- `scaleY(~scaleY=float)`
+- `translateX(~translateX=float)`
+- `translateY(~translateY=float)`
+- `skewX(~skewX=angle)`
+- `skewY(~skewY=angle)`
+
+##### Transform Examples
+
+###### Transform with multiple transform
+
+```reason
+Style.(style(
+  ~transform=[|
+    perspective(~perspective=1000.),
+    rotateX(~rotateX=20.->deg),
+    rotateZ(~rotateZ=0.5->rad),
+    scale(~scale=0.95),
+  |],
+  ()
+))
+```
+
+###### Transform with an animated value
+
+```reason
+Style.(style(
+  ~transform=[|
+    rotateY(
+      ~rotateY=
+        Animated.Interpolation.(
+          scrollYAnimatedValue->interpolate(
+            config(
+              ~inputRange=[|0., 200.|],
+              ~outputRange=
+                [|"-10deg", "-14deg"|]->fromStringArray,
+              ~extrapolateLeft=`clamp,
+              ~extrapolate=`identity,
+              ~extrapolateRight=`extend,
+              (),
+            ),
+          )
+        )
+        ->Animated.StyleProp.angle,
+    ),
+    scale(
+      ~scale=
+        Animated.Interpolation.(
+          scrollYAnimatedValue->interpolate(
+            config(
+              ~inputRange=[|0., 200.|],
+              ~outputRange=[|0.8, 0.75|]->fromFloatArray,
+              (),
+            ),
+          )
+        )
+        ->Animated.StyleProp.float,
+    ),
+  |],
+  (),
+))
+```
+
+If you need something unsupported by your platform, you can use
```

Comment:
should be:
If you need something supported by your platform, you can use
or perhaps:
If you need some other transform supported by your platform, you can use

Reply:
I used `If you need something unsupported by this binding`